apps/vaporgui/VaporTable.cpp

```diff
@@ -381,6 +381,7 @@ void VaporTable::setVerticalHeader(std::vector<std::string> header) {
 	_table->setVerticalHeaderLabels(list);
 	_table->resizeRowsToContents();
 	_table->verticalHeader()->setResizeMode(QHeaderView::Stretch);
+    _table->verticalHeader()->setMaximumWidth(100);
```

Comment:
You're hard-coding the header width (whatever that is) for all instances of the VaporTable. Isn't that likely to break something somewhere else? Wouldn't it be better to add a wrapper method to the VaporTable class and let the user of the class make this change if needed?

Reply:
Maybe we can talk about this.  I think that in some situations it's reasonable to hard code values (IE TabManager).  I'm pretty certain that this isn't going to break anything.